src/components/DatasetDirectionalModal.jsx

```diff
@@ -0,0 +1,180 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import { Link } from 'react-router-dom';
+import Dialog from '@material-ui/core/Dialog';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import DialogContent from '@material-ui/core/DialogContent';
+import IconButton from '@material-ui/core/IconButton';
+import CloseIcon from '@material-ui/icons/Close';
+import Button from '@material-ui/core/Button';
+
+const styles = theme => ({
+  title: {
+    color: theme.palette.primary.main,
+    fontSize: '54px',
+    lineHeight: '66px',
+    paddingBottom: theme.spacing.unit * 8,
+  },
+  dialogTitle: {
+    borderBottom: `1px solid ${theme.palette.divider}`,
+    display: 'flex',
+    justifyContent: 'space-between',
+    margin: 0,
+    padding: theme.spacing.unit * 3,
+  },
+  dialogContent: {
+    paddingBottom: theme.spacing.unit * 4,
+    paddingTop: theme.spacing.unit * 4,
+    fontFamily: [
+      'Montserrat',
+      'BlinkMacSystemFont',
+      '"Segoe UI"',
+      'Roboto',
+      '"Helvetica Neue"',
+      'Arial',
+      'sans-serif',
+      '"Apple Color Emoji"',
+      '"Segoe UI Emoji"',
+      '"Segoe UI Symbol"',
+    ].join(','),
+    fontSize: '20px',
+    fontWeight: '500',
+  },
+  dialogActions: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    paddingBottom: theme.spacing.unit * 2,
+    paddingTop: theme.spacing.unit * 2,
+    minWidth: '600px',
+  },
+  actionButtons: {
+    color: theme.palette.primary.contrastText,
+    textDecoration: 'none',
+  },
+  actionButtonsError: {
+    backgroundColor: theme.palette.error.main,
+    color: theme.palette.primary.contrastText,
+    float: 'right',
+    textDecoration: 'none',
+  },
+  closetitle: {
+    display: 'inline-block',
+  },
+  closeButton: {
+    position: 'absolute',
+    right: theme.spacing.unit * 1.5,
+    top: theme.spacing.unit * 2,
+  },
+});
+
+export class DatasetDirectionalModal extends React.PureComponent {
+  static propTypes = {
+    classes: PropTypes.object.isRequired,
+    createdDataset: PropTypes.object.isRequired,
+    success: PropTypes.bool.isRequired,
+  };
+
+  state = {
+    open: true,
+  };
+
+  handleClose = () => {
+    this.setState({ open: false });
+  };
+
+  render() {
+    const { classes, createdDataset, success } = this.props;
+    const { open } = this.state;
+    return (
+      <div id="dataset-modal">
+        {success ? (
+          <Dialog
+            onClose={this.handleClose}
+            aria-labelledby="customized-dialog-title"
+            open={open}
+            maxWidth={false}
+          >
+            <DialogTitle className={classes.dialogTitle} id="customized-dialog-title">
+              <div className={classes.closetitle}>Where to next?</div>
```

Comment:
nit: I know I suggested this text so feel free to throw something at me, but the use of next twice in this modal seems excessive now that I see it -- let's just have this be "Where to?"